packages/website/tsconfig.json

```diff
@@ -4,7 +4,11 @@
     "allowJs": true,
     "checkJs": true,
     "target": "esnext",
-    "lib": ["ESNext", "DOM", "DOM.Iterable"],
+    "lib": [
```

Comment:
I"m glad you checked this in. My local kept reformatting this too but I wasn't sure if it was something weird on my setup. 🙏

Reply:
I've actually been reverting this change for awhile, and pushing this up was a total accident.  I'm glad I'm not the only one!  I will leave it in there.